Lib/pydoc.py

```diff
@@ -2796,6 +2779,10 @@ class BadUsage(Exception): pass
                 hostname = val
 
         if start_server:
+            warnings.warn("WARNING: The web server is accessible by other users "
+                          "on this computer. This will potentially make all "
+                          "content accessible by you visible to anyone else "
+                          "sharing the machine.", category=RuntimeWarning)
```

Comment:
Wait. Is there another way to access the filesystem from a browser, after getfile is removed?

Reply:
AFAIK no. However, pydoc currently works by calling inspect on files it sees in path, and this may reveal private code as Marc-Andre Lemburg pointed out on the bpo. I will try the random url token he suggested via ``secrets.token_urlsafe`` to see if it helps.

Reply:
I removed the warning since it should now be secure with the token/secret.